python-sdk/tests/benchmark/Dockerfile

```diff
@@ -24,9 +25,11 @@ RUN apt install -y vim
 RUN apt install -y curl
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && apt-get update -y && apt-get install google-cloud-cli -y
 
-# Install Astro SDK dependencies
+RUN apt install -y libpq-dev python3-dev
 
+# Install Astro SDK dependencies
 COPY . .
+COPY python-sdk .
```

Comment:
Maybe it would make more sense to set the WORKDIR to python-sdk here? Is there anything in the base directory we really need?